pseudocode.py

```diff
@@ -0,0 +1,129 @@
+'''
+Aerolyzer - Pseudocode
+Feburary 10, 2017
+
+
+Image Restrictions:
+
+Throughout the upload process, images will be required to meet various
+criteria beforee being accepted for aerosol content analysis. To accomplish this
+requirement, a mulititude of functions will be created, each aimed at verifiying
+that a certain restriction has been met. To ensure the simplicity of Aerolyzer's
+initial development, as well as future adaptations, information regarding image criteria
+will be stored in a single file to be referenced by the various restriction functions
+outlined below.
+'''
+
+'''
+Image Criteria File
+Purpose:        The purpose of this file is to keep all image criteria specifications located in
+                one place, for simplicity's sake.
+Inputs:         None
+Outputs:        None
+Returns:        None
+Assumptions:    These values will be populated by a developer.
+'''
+accepted_file_types = ["JPEG", "JPG", "PNG"]
+accepted_mobile_devices = ["i5", "i5s", "i6", "i6s", "i7"]
+img_max_size = 200;
+img_width_min = xxx;
+img_height_min = xxx;
+
+'''
+Restriction:    The image must be from an accepted mobile device. 
+Purpose:        The purpose of this function is to determine whether or not the device the
+                image was taken on is an accepted mobile device.
+Inputs:         EXIF data, list of accepted mobile devices
+Outputs:        None
+Returns:        True, False
+Assumptions:    N/A
+'''
+def (**exif_data, accepted_mobile_devices):
```

Comment:
2. The reason behind using ** was simply because I'm unfamiliar with python and the dictionary type,  so I tried googling how to pass dictionary as a parameter to a function and found the following site: http://stackoverflow.com/questions/21986194/how-to-pass-dictionary-items-as-function-arguments-in-python
I saw that they were using ** so I just went with it, but I will change it to just say exif_data.

3. I must have mixed these up during our last meeting, I thought we were using underscores for functions and variable names. But I will change all variables to camelCase accordingly.